modules/failure-handler/src/main/java/org/apache/ignite/internal/failure/FailureProcessor.java

```diff
@@ -145,4 +154,14 @@ private static boolean failureTypeIgnored(FailureContext failureCtx, FailureHand
         return handler instanceof AbstractFailureHandler
                 && ((AbstractFailureHandler) handler).ignoredFailureTypes().contains(failureCtx.type());
     }
+
+    /**
+     * Set FailHander interceptor to provide ability tщ test scenarios related to fail handler.
+     *
+     * @param interceptor Interceptor of fails.
+     */
+    @TestOnly
+    public synchronized void setInterceptor(@Nullable FailureHandler interceptor) {
```

Comment:
does it make to add TODO here to remove this method when failure handler becomes configurable?